tools/push_api_gh_pages.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ev
+
+GH_REPO="@github.com/elopio/snapcraft.git"
```

Comment:
@sergiusens done. But then you need to update the token in the travis settings, with permissions to repo and user.